src/coreclr/jit/emit.cpp

```diff
@@ -913,6 +913,9 @@ insGroup* emitter::emitSavIG(bool emitAdd)
         }
 
         emitAlignLast = last;
+        // Point to the first instruction of most recent
+        // align instruction(s) added.
+        emitAlignLastGroup = list;
```

Comment:
It looks like the emitAlignList is in the reverse order of the emitCurIGAlignList loop, due to inserting in reverse order. So, does `list` actually represent the earliest-in-program-order `align` instruction, or the last one? I presume the code here wants it to be the "earliest". If that's indeed so, can you explain why?

Reply:
Hmmm... It looks like `emitLoopAlign` creates the list in inverse-of-program-order, so this actually reverses that. Worth a comment.

Reply:
Yes - It is essentially in the same way we create `emitCurIGjmpList`. I will add a comment.